.github/workflows/native-component-list.yml

```diff
@@ -0,0 +1,51 @@
+name: Native Component List app
+
+on:
+  workflow_dispatch: {}
+  pull_request:
+    branches: [master]
+    paths:
+      - .github/workflows/native-component-list.yml
+      - apps/native-component-list/**
+      - yarn.lock
+  push:
+    branches: [master]
+    paths:
+      - .github/workflows/native-component-list.yml
+      - apps/native-component-list/**
+      - yarn.lock
+
+jobs:
+  build:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
+      - uses: actions/setup-node@v2-beta
+        with:
+          node-version: '12'
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+      - uses: actions/cache@v1
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+      - run: yarn install --frozen-lockfile
+      - run: yarn tsc
```

Comment:
I wonder if it's possible to encapsulate some common steps from every action into one? 🤔 
I've found sth like this: https://github.blog/changelog/2020-08-07-github-actions-composite-run-steps/
What would you say? 🤔

Reply:
If we decide to do so let's move it to a separate PR — I basically copied `home.yml`, so keeping it the same here and there for now will make it consistent.